src/encoded/schemas/experiment.json

```diff
@@ -234,6 +234,15 @@
                 "organoid"
             ]
         },
+        "experiment_class": {
+            "title": "Experiment class",
+            "description": "The categorization of the experiment.",
+            "type": "string",
+            "enum" : [
+                "ground level annotation",
```

Comment:
I'm not sold on this enum value though I don't have a viable alternate to offer at the moment.